00_basics/py_dimensions.ipynb

```diff
@@ -0,0 +1,740 @@
+{
```

Comment:
<p>btw. is it not a convention in python to have spaces between operators and operands?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/capital-G/musikinformatik-sose2021/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>